Dockerfile.repo

```diff
@@ -90,14 +89,12 @@ WORKDIR $ACAS_HOME
 
 ENV R_LIBS $ACAS_HOME/r_libs
 RUN mkdir $ACAS_HOME/r_libs
-ENV PATH /usr/pgsql-13/bin/:$PATH
```

Comment:
No longer required because of the way we are installing Postgres from yum rpms which configure paths correctly.